src/hypothesis/internal/reflection.py

```diff
@@ -360,14 +360,15 @@ def source_exec_as_module(source):
 
 def accept(%(funcname)s):
     def %(name)s(%(argspec)s):
+        '''%(docstring)s'''
```

Comment:
Sorry, I should have caught this before.

I don't think this is a good way to do this and think it'll almost certainly get things mangled. Assigning the \_\_doc\_\_ property works (I think we do that elsewhere?) and should probably be done instead.